src/main/java/teammates/logic/api/GateKeeper.java

```diff
@@ -256,20 +277,23 @@ public void verifyAccessible(InstructorAttributes instructor, FeedbackSessionAtt
         }
     }
 
-    public void verifyAccessible(InstructorAttributes instructor, FeedbackSessionAttributes feedbacksession,
+    /**
+     * Verifies that the specified instructor can access the specified feedback session.
```

Comment:
`Verifies that the specified instructor has certain privilege for a section in the specified feedback session.`?